mobile/library/cc/engine_builder.cc

```diff
@@ -450,13 +459,16 @@ std::unique_ptr<envoy::config::bootstrap::v3::Bootstrap> EngineBuilder::generate
 
   for (auto filter = native_filter_chain_.rbegin(); filter != native_filter_chain_.rend();
        ++filter) {
-#ifdef ENVOY_ENABLE_YAML
     auto* native_filter = hcm->add_http_filters();
+#ifdef ENVOY_ENABLE_YAML
     native_filter->set_name((*filter).name_);
```

Comment:
nit: I believe the `(*filter)` can be replaced with `filter->`

Reply:
good point - will do in the follow up